src/leveled.app.src

```diff
@@ -1,7 +1,7 @@
 {application, leveled,
  [
   {description, "Key Value store based on LSM-Tree and designed for larger values"},
-  {vsn, "0.9.20"},
+  {vsn, "0.9.21"},
```

Comment:
Since you removed the "skip" strategy, one could wonder if this is a ptach or a minor update. I would go for 0.10.0... but that's fully up to you.

Reply:
It was more of a confusion over whether the strategy was called 'skip' or 'recovr', I think.